app/views/contacts/_show.html.slim

```diff
@@ -0,0 +1,24 @@
+tr
```

Comment:
If you pass `, contact: @department.contact` to the partial
the partial will only need to reference `contact.first_name` instead of `entity.contact.first_name`